### PR TITLE
Issue 487 tooltip design update

### PR DESCRIPTION
**Pull request by mahmoud-zakria-sociomantic**
#487

### CODE REVIEW

File: src/Tooltip/tooltip.css [hunk 1]
```diff
@@ -338,22 +497,174 @@
...
+    }
+    &.position__top,
+    &.position__topLeft,
+    &.position__topRight
+    {
+      .tooltip
+      {
+        &::after
+        {
+          border-top-color: var( --PC-RED );
```
Comment (conor-cafferkey-sociomantic) [inline src/Tooltip/tooltip.css]: Why set this individually for each position? I think you can just do `tooltip::after { border-color: ... }` one time and be done.... right? 🤔

File: src/Tooltip/tooltip.css [hunk 2]
```diff
@@ -135,201 +199,296 @@
...
+
 .position__top,
 .position__bottom
 {
     .tooltipContainer
     {
         left:   50%;
-        width:  calc( var( --tooltipMaxWidth ) * 2 );
-        height: var( --tooltipMaxHeight );
+        transform: translate3d( -50%, 0, 0 );
```
Comment (conor-cafferkey-sociomantic) [inline src/Tooltip/tooltip.css]: Since we’re only `translate`-ing in one axis here let’s just use `translateX`

File: src/Tooltip/tooltip.css [hunk 3]
```diff
@@ -135,201 +199,296 @@
...
             left:   100%;
         }
+        &::after
+        {
+          border-right-color: transparent;
+          right: 0;
+          transform: translate3d( calc( 1.75 * var( --tooltipOffset ) ), -50%, 0);
+        }
+    }
+    &Top
```
Comment (conor-cafferkey-sociomantic) [inline src/Tooltip/tooltip.css]: If we want to use this type of nesting style to build classname selectors here then – for consistency – we should also use it elsewhere this file (e.g. for `topLeft`, `topRight` `bottomLeft`, `bottomRight`).

(I gotta say I’m generally not really a fan of this style tho. One of the creators of Sass has even said that he [sincerely regrets adding it](https://github.com/w3c/csswg-drafts/issues/2701#issuecomment-395572904). That said, I can kinda see the value of it in this case so I fine with using it for this component.)

File: src/Tooltip/tooltip.css (hunk 1 above)
Comment (mahmoud-zakria-sociomantic) [reply src/Tooltip/tooltip.css]: Done

File: package.json [hunk 4]
```diff
@@ -112,6 +112,7 @@
     "postcss-custom-selectors": "^4.0.1",
     "postcss-extend": "^1.0.5",
     "postcss-font-variant": "^3.0.0",
+    "postcss-hexrgba": "^1.0.1",
```
[the remark this answers is not in the document]
Comment (mahmoud-zakria-sociomantic) [reply package.json]: Yeah it was added for convert colors for box-shadow from `hex` to `rgb` but then found `color()` and used it.

[the remark this answers is not in the document]
Comment (mahmoud-zakria-sociomantic) [reply package.json]: Done.

**Pull request merged**